mgmt/rest/v1/task.go

```diff
@@ -45,11 +46,23 @@ var (
 )
 
 func (s *apiV1) addTask(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
-	task, err := core.CreateTaskFromContent(r.Body, nil, s.taskManager.CreateTask)
-	if err != nil {
-		rbody.Write(500, rbody.FromError(err), w)
+	task, errs := core.CreateTaskFromContent(r.Body, nil, s.taskManager.CreateTask)
+	if errs != nil && len(errs.Errors()) != 0 {
+		testSerrors := make([]serror.SnapError, 0)
+
+		for _, e := range errs.Errors() {
+			testSerrors = append(testSerrors, serror.New(errors.New(e.Error()), e.Fields()))
+		}
+
+		log.WithFields(log.Fields{
+			"_file":     "rest/v1/task.go,",
+			"_function": "addTask",
+			"error":     errors.New(testSerrors[0].Error()),
+		}).Error("error adding task")
+		rbody.Write(500, rbody.FromSnapErrors(testSerrors), w)
```

Comment:
@Alrim42, could you verify if all that added code (lines 51-61) is needed? I think it should be just one line `rbody.Write(500, rbody.FromSnapErrors(errs.Errors()), w)`, but please check it.